pulse/src/main/scala/io/vamp/pulse/PulseBootstrap.scala

```diff
@@ -13,9 +13,22 @@ object PulseBootstrap {
 class PulseBootstrap extends ActorBootstrap with PulseNotificationProvider {
 
```

Comment:
We should maybe look into a way to add a separate bootstrap, but it's fine for the time being